tests/src/ondevice_tests/crashdump_test.cpp

```
@@ -0,0 +1,61 @@
+#include <atomic>
+#include <catch2/catch_all.hpp>
+#include <chrono>
+#include <condition_variable>
+#include <mutex>
+#include <thread>
+
+#include "depthai/depthai.hpp"
+#include "utility/Environment.hpp"
+
+TEST_CASE("Crashdump callback is invoked on device crash") {
+    dai::utility::setEnv("DEPTHAI_CRASHDUMP", "0");  // don't save nor upload crash dump
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)